packages/infra/lib/surescripts-stack.ts

```diff
@@ -0,0 +1,486 @@
+import { Duration, NestedStack, NestedStackProps } from "aws-cdk-lib";
+import { SnsAction } from "aws-cdk-lib/aws-cloudwatch-actions";
+import * as ec2 from "aws-cdk-lib/aws-ec2";
+import { Function as Lambda } from "aws-cdk-lib/aws-lambda";
+import { SqsEventSource } from "aws-cdk-lib/aws-lambda-event-sources";
+import * as s3 from "aws-cdk-lib/aws-s3";
+import { Queue } from "aws-cdk-lib/aws-sqs";
+import { Construct } from "constructs";
+import { EnvConfig } from "../config/env-config";
+import { EnvType } from "./env-type";
+import { createLambda } from "./shared/lambda";
+import { LambdaLayers } from "./shared/lambda-layers";
+import { QueueAndLambdaSettings } from "./shared/settings";
+import { createQueue } from "./shared/sqs";
+
+const waitTimePatientSync = Duration.seconds(10); // 6 patients/min
+const waitTimeRefreshBundle = Duration.seconds(0); // No limit
+
+function settings(): {
+  connectSftp: QueueAndLambdaSettings;
+  synchronizeSftp: QueueAndLambdaSettings;
+  generateRequest: QueueAndLambdaSettings;
+  parseVerification: QueueAndLambdaSettings;
+  parseResponse: QueueAndLambdaSettings;
+} {
+  const connectSftp: QueueAndLambdaSettings = {
+    name: "SurescriptsConnectSftp",
+    entry: "surescripts-connect-sftp",
+    lambda: {
+      memory: 512,
+      timeout: Duration.seconds(30),
+    },
+    queue: {
+      alarmMaxAgeOfOldestMessage: Duration.minutes(10),
+      maxMessageCountAlarmThreshold: 100,
+      maxReceiveCount: 3,
+      visibilityTimeout: Duration.seconds(10 * 2 + 1),
+      createRetryLambda: false,
+    },
```

Comment:
think we're not using these queue settings right? in that case, delete!

Reply:
Added a new `LambdaSettings` interface for this so requires a fresh look (LambdaAndQueue requires both to be defined).